### DISCUSSION TITLE
Background Menu Video

### DISCUSSION THREAD

**Discussion by HazmatKat** [pukmajster/funky#20] [Ideas]
This is probably a long shot, but if there was some way to make it shuffle the background video of the in-game menu. I know that's something you have to do manually and it doesn't work with .vpk so I am very unsure how hard that would be to implement.

**Comment by pukmajster** [comment]
If the .vpk comes with the video file inside, then yes, it's doable. I do, however, have to re-implement VPK extraction first in order to be able to do this. It'll happen someday :D
